doc/source/getting_started/write_documentation.rst

```diff
@@ -178,55 +194,55 @@ Dynamic examples
 ----------------
 
 Dynamic examples are based on Python files and must be able to run in under three minutes.
+In the PyMAPDL repository, they are in the `examples <pymapdl_examples_>`_ directory.
 
-In the PyMAPD repository, they are in the `examples <pymapdl_examples_>`_ directory.
-
-.. vale off
-
-Example: `2d_plate_with_a_hole.py <pymapdl_2d_plate_with_a_hole_>`_
-.. vale on
-
-Here is a link to this dynamic example: 
-`MAPDL 2D Plane Stress Concentration Analysis <pymapdl_doc_2d_plate_with_a_hole_>`_
-
-When an example is executed, **Total running time of the script** appears at the end of
-the document.
+One example of a dynamic examples is the `MAPDL 2D Plane Stress Concentration Analysis <pymapdl_2d_plate_with_a_hole_>`_
+example.
+The source code can be found in this GitHub `link <pymapdl_2d_plate_with_a_hole_>`_. 
```

Comment:
Another case where "link" isn't descriptive text. It's the same as using "here". You should replace with the name of the destination.